contrib/README.md

```diff
@@ -6,12 +6,13 @@ This project follows [Semantic Versioning 2.0](https://semver.org/). In general,
 
 ### Publishing a New SDK version
 ```bash
-# run bump_version.bash with either patch, minor, or major
-./bump_version.bash <snapshot|beta|patch|minor|major>
-
-# send an email detailing the changes to the https://groups.google.com/d/forum/mongodb-stitch-announce mailing list
+# run bump_version.bash with either patch, minor, or major followed by the JIRA ticket number (you may omit the STITCH keyword if you would like).
+./bump_version.bash <snapshot|beta|patch|minor|major> <STITCH-1234|1234>
 ```
 
+* send an email detailing the changes to the https://groups.google.com/d/forum/mongodb-stitch-announce mailing list
+* go to [Android SDK](https://github.com/mongodb/stitch-android-sdk/pulls) and request a reviewer on the pull request (mandatory) before merging and deleting the release branch
```

Comment:
[q] does `bump_version.bash` not do this automatically with the new `hub` command?